### ISSUE TITLE
how to generate doc for an helm chart having multiple values files

### ISSUE THREAD

**Issue by NatarajanR** [norwoodj/helm-docs#119]
how to generate doc for an helm chart having multiple values files? For example, refer https://github.com/temporalio/helm-charts chart. It would have a default values.yaml and then a directory called values, inside which several values files will be there.

**Comment by sc250024** [comment]
> how to generate doc for an helm chart having multiple values files? For example, refer https://github.com/temporalio/helm-charts chart. It would have a default values.yaml and then a directory called values, inside which several values files will be there.

I'm not sure this is a use case that should be supported. While having multiple values files is useful to see what settings are useful in different scenarios, there's only ever one canonical file that the chart uses: the values.yaml file itself.

**Comment by norwoodj** [comment]
Yeah, this is a pretty niche case and I don't think it should be supported by this tool
